routers/users.py

```
@@ -0,0 +1,46 @@
+from fastapi import APIRouter
+
+from schemas.commons import UserId
+
+router = APIRouter(
+    tags=["USERS"],
+)
+
+
+# signup
+@router.post("/users")
+async def create_user(user: dict):
```

Ruff (0.14.11)

Unused function argument: `user`

(ARG001)